packages/front/src/lib/components/Documents/Documents.controller.ts

```diff
@@ -74,22 +84,47 @@ export class DocumentsController {
     async _getEstablishmentDocuments(establishment) {
         const association = currentAssociation.value;
         if (!association) return [];
-        const estabDocsPromise = establishmentService.getDocuments(establishment.siret);
+        const estabDocsPromise: Promise<LabeledDoc[]> = establishmentService
+            .getDocuments(establishment.siret)
+            .then(docs => docs.map(doc => ({ ...doc, showByDefault: true } as LabeledDoc)));
         const assoDocsPromise = associationService
             .getDocuments(association.rna || association.siren)
-            .then(docs => docs.filter(doc => !doc.__meta__.siret || doc.__meta__.siret === establishment.siret));
+            .then(docs => documentService.labelAssoDocsBySiret(docs, establishment.siret));
         const [estabDocs, assoDocs] = await Promise.all([estabDocsPromise, assoDocsPromise]);
         return this._removeDuplicates([...estabDocs, ...assoDocs]);
     }
 
-    _organizeDocuments(miscDocs: DocumentEntity[]) {
-        const assoDocs: DocumentEntity[] = [];
-        const estabDocs: DocumentEntity[] = [];
+    _organizeDocuments(miscDocs: LabeledDoc[]): GroupedDocs {
+        const fullAssoDocs: LabeledDoc[] = [];
+        const fullEstabDocs: LabeledDoc[] = [];
         for (const doc of miscDocs) {
-            if (["Le Compte Asso", "Dauphin"].includes(doc.provider)) estabDocs.push(doc);
-            if (["RNA", "Avis de Situation Insee"].includes(doc.provider)) assoDocs.push(doc);
+            if (["Le Compte Asso", "Dauphin"].includes(doc.provider)) fullEstabDocs.push(doc);
+            if (["RNA", "Avis de Situation Insee"].includes(doc.provider)) fullAssoDocs.push(doc);
+            // TODO where to put others ?
         }
-        return { assoDocs, estabDocs, some: !!(assoDocs.length + estabDocs.length) };
+        sortLabeledDocs(fullAssoDocs);
+        sortLabeledDocs(fullEstabDocs);
+        const assoDocs = fullAssoDocs.filter(doc => doc.showByDefault);
+        const estabDocs = fullEstabDocs.filter(doc => doc.showByDefault);
+        const moreAssoDocs = fullAssoDocs.filter(doc => !doc.showByDefault);
+        const moreEstabDocs = fullEstabDocs.filter(doc => !doc.showByDefault);
+        return {
+            moreAssoDocs,
+            moreEstabDocs,
+            assoDocs,
+            estabDocs,
+            someAsso: !!fullAssoDocs.length,
+            someEstab: !!fullEstabDocs.length,
+            some: !!(assoDocs.length + estabDocs.length),
+            fullSome: !!(fullAssoDocs.length + fullEstabDocs.length),
+        };
+    }
+
+    docsToShow(docs: GroupedDocs, structureType: "association" | "establishment", more = false) {
```

Comment:
pas fan du troisième argument comme ça, c'est pas très compréhensible lors de l'appel.

```suggestion
    docsToShow(docs: GroupedDocs, structureType: "association" | "establishment", options: { full = boolean } {
```

```suggestion
    docsToShow(docs: GroupedDocs, structureType: "association" | "establishment", options: { more: boolean } {
```

En exécution ça donne `controller.docsToShow(documents, "association", {full: true))` je trouve ça plus compréhensible à la lecture, ça ne nécessite pas de lire la définition de `docsToShow`

Reply:
Et j'irai même encore plus loin, le premier paramètre est toujours le même (`documents`) donc je le supprimerai. Et je créerai deux sous méthode assoDocsToShow et estabDocsToShow afin de ne pas avoir à écrire à la main "association" et "establishment" et éviter les erreurs de saisies

Reply:
J'aime pas trop le fait de créer un objet qui n'a qu'un paramètre comme ça :/
Pour les deux sous-méthodes j'aime pas trop la duplication mais j'avais hésité donc ok.
Pour le fait que le premier paramètre soit toujours le même, je peux pas trop bouger sans encore refacto plein de choses parce que là `docs` qui est redonné c'est le résultat de la promesse donc le contrôleur le fabrique mais ce n'est pas un attribut facile d'accès pour autant

Reply:
Ok. Pas forcément besoin de faire la duplication, mais au moins utiliser une enum plutôt que recopier "association" et "establishment", non ?

Reply:
Ça me paraît plus lourd. Mais pour la même raison que le `#key` je n'ai plus besoin de dynamisme donc j'ai juste enlevé le fait de passer par une fonction

Reply:
ok !